packages/common/mui/src/lib/data-table-cells/multi-value-cell/multi-value-cell.tsx

```diff
@@ -47,7 +54,8 @@ export const MultiValueCell = ({ cell, row, column }): JSX.Element => {
                     sx={{
                       marginBottom: '0px',
                       width: '100%',
-                      fontWeight: row.original.currency === 'Total' ? 600 : 500,
+                      fontWeight:
+                        isTotalRow || row.original.isEarningsRow ? 600 : 500,
                       color:
```

Comment:
Earnings font size is wrong:
![image](https://github.com/notional-finance/notional-monorepo/assets/977434/0750fb95-7a87-4e34-80db-2e85f943b991)